deploy/example_catalog/cr-app-cdh632-1.1cm.json

```diff
@@ -0,0 +1,421 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "cdh632"
+    },
+
+    "spec" : {
+        "systemdRequired" : true,
+        "defaultPersistDirs" : ["/opt", "/var", "/data"],
```

Comment:
do we need to persist these 3 directories for all roles?

Reply:
@swamibluedata /data is required to be persisted only for HDFS. accordingly removed /data from defaultPersistDirs and added /data in persistDirs keys for primary, standby, arbiter and worker roles. These roles use HDFS service.